connectivity/FEATURE_BLE/source/generic/GapImpl.h

```diff
@@ -535,6 +538,20 @@ class Gap :
 
     void process_legacy_scan_timeout();
 
+    /* Implement PrivateAddressController::EventHandler */
+private:
+    void on_resolvable_private_addresses_generated(const address_t &address) final;
+
+    void on_non_resolvable_private_addresses_generated(const address_t &address) final;
+
+    void on_private_address_generated(bool connectable);
+
+    void on_address_resolution_completion(
```

Comment:
`_completed` to be consistent with the 3 function names above but I would be happy with just `_complete`

Reply:
Fixed.